scribble-lib/scriblib/autobib.rkt

```diff
@@ -563,6 +565,27 @@
          #:degree [degree "PhD"])
   @elem{@to-string[degree] dissertation, @to-string[org]})
 
+(define (book-chapter-location
+         location
+         #:pages [pages #f]
+         #:series [series #f]
+         #:volume [volume #f]
+         #:publisher [publisher #f])
+  (let* ([s @elem{In @italic{@elem{@|location|}}}]
```

Comment:
The contract on `location` is `any/c`, so I think this should be `@italic{@to-string[location]}`

... maybe the contract should be `string?` because that's what this line is assuming?